src/main/java/org/openrewrite/java/migrate/ReplaceAWTGetPeerMethod.java

```diff
@@ -35,8 +32,14 @@
 @EqualsAndHashCode(callSuper = false)
 class ReplaceAWTGetPeerMethod extends Recipe {
 
-    // Fields configurable for tests
+    @Option(displayName = "Method pattern to replace",
```

Comment:
@timtebeek with this change, it seems like envision this recipe could be reusable. If that's the case, then maybe it makes sense to rename the class name to something more generic since `ReplaceAWTGetPeerMethod` gives the impression it is only applicable to the getPeer() method

Reply:
Oh no it should not be reusable; but we needed these options to be able to test the recipe, and figured document the options if they are going to show up in the docs anyway.

Reply:
Ahh okay, I see what's going on.